Mixpanel.xcodeproj/project.pbxproj

```diff
@@ -1026,6 +939,7 @@
 		E11594861CFF1491007F8B4F /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = YES;
```

Comment:
AFAIK, this is set in order to not allow the compiler or linker to let developers access APIs they shouldn't access in appex (such as UIApplication.shared). In our case because we are accessing these APIs (implicitly) only in the main app and not in the appex (which we figure out during runtime), we are fine. However, i do prefer to be less strict as there may be unknowns and I've seen that this flag is automatically turned on for Appex apps anyway, so i'll set it as NO for now.